### PR TITLE
replace 'Home' with dynamic homeType

### PR DESCRIPTION
**Pull request by jerminatorhits**
## Description
Why did you write this code?
Replace hardcoded 'Home' to dynamic 'homeType' so that Hotel Rooms don't display as `Amy's Home`, but rather `Amy's Hotel Room`
Requires https://github.com/thebeetoken/beenest-backend/pull/682

## How to Test
- [ ] Click 'Request to Book'
- [ ] Verify the Trip details shows host's name + homeType

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/components/routes/Booking/BookingQuote/BookingQuote.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const BookingQuote = ({ booking, currency, fromBee }: Props) => {
             {host.firstName}
```
Comment (tc) [inline src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: how about we show:
## Listing Title
[small] homeType
[small] City

Comment (jerminatorhits) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: listing titles can get very long

image

Comment (kcvan) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: we can truncate the listing title to a set number of letters if we want the title

Comment (tc) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: Can we try this:
Replace Trip Details with Listing title and go one size smaller.
Replace "31 Nights at Listing Title" with "homeType in Los Angeles"

Comment (tc) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: screen shot 2019-01-24 at 6 13 03 pm

Comment (tc) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: screen shot 2019-01-24 at 6 19 50 pm

Comment (tc) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: screen shot 2019-01-24 at 6 21 04 pm

Comment (kcvan) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: i like the idea of replacing the "31 nights" thing.  i think we should still truncate just in case since we dont limit the amount of characters for the listing title for non-beenest providers

Comment (tc) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: attempt at sketch edit :)
https://projects.plantapp.io/p/18560/13/DBB46102-0CA3-4E10-B89A-B13EB95A5C4F?r=13
screen shot 2019-01-24 at 6 34 46 pm

Comment (tc) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: yea we can truncate text with css to two lines?

Comment (jerminatorhits) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: doesn't seem like there's an easy way to truncate to 2 lines.
https://stackoverflow.com/questions/5269713/css-ellipsis-on-second-line

@kcvan how long do the titles get? I think 3 lines might be passable.

File: src/components/routes/Booking/BookingOptions/SelectPaymentOption/SelectPaymentOption.container.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ const SelectPaymentOptionContainerTablet = styled(SelectPaymentOptionContainerMo
 
 
     .select-payment-quote-desktop {
-      height: 440px;
+      height: auto;
```
Comment (tc) [inline src/components/routes/Booking/BookingOptions/SelectPaymentOption/SelectPaymentOption.container.ts]: nice, removing the fixed height is good. is auto the default? 
would that make a long title work? it'll just make the box height taller?

File: src/components/routes/Booking/BookingQuote/BookingQuote.tsx (hunk 1 above)
Comment (kcvan) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: @jerminatorhits if it's not uploaded from us, I'm not sure how long the titles can get, since we don't control the listing title length that come from other providers

File: src/components/routes/Booking/BookingOptions/SelectPaymentOption/SelectPaymentOption.container.ts (hunk 2 above)
Comment (kcvan) [reply src/components/routes/Booking/BookingOptions/SelectPaymentOption/SelectPaymentOption.container.ts]: `height: auto` is default for height.  let me take a look tomorrow to make sure it works well on all screens

File: src/components/routes/Booking/BookingQuote/BookingQuote.tsx (hunk 1 above)
Comment (jerminatorhits) [reply src/components/routes/Booking/BookingQuote/BookingQuote.tsx]: ahh i see. Long titles will be an issue across the app (Listing Cards, Booking Cards, mobile views, etc.) If we solve it here, we can do the same in other components!

**Comment by tc** [comment]
👍

File: src/components/routes/Booking/BookingOptions/SelectPaymentOption/SelectPaymentOption.container.ts (hunk 2 above)
Comment (jerminatorhits) [reply src/components/routes/Booking/BookingOptions/SelectPaymentOption/SelectPaymentOption.container.ts]: yeah. I thought I saw `height: 100%` being declared in mobile but I read it wrong. REMOVED

**Comment by tc** [comment]
lgtm!
